juniper_official/Solutions/RPM/rpm-probe-netconf.rule

```diff
@@ -0,0 +1,187 @@
+                description "RPM iAgent sensor to collect telemetry data from network device";
+                iAgent {
+                    file RPMprobe.yml;
+                    table RPMprobeTable;
+                    frequency 60s;
+                }
+            }
+            /*
+             * Fields defined using sensor path. Map the longer sensor names
+             * to the shorter field names used in the rules.
+             */
+            field loss-percentage {
+                sensor rpm-probe {
+                    path loss_percentage;
+                }
+                type integer;
+                description "RPM probe loss percentage";
+            }
+            field owner {
+                sensor rpm-probe {
+                    path owner;
+                }
+                type string;
+                description "RPM probe owner";
+            }
+            field probe-rt-delay {
+                sensor rpm-probe {
+                    path probe_rt_ave_delay;
+                }
+                type integer;
+                description "RPM probe rt delay";
+            }
+            field probe-rt-delay-dt {
+                formula {
+                    anomaly-detection {
+                        algorithm k-means;
+                        learning-period 7d;
+                        pattern-periodicity 1d,1h;
+                        field-name "$probe-rt-delay";
+                    }
+                }
+                type integer;
+            }
+            field test-name {
+                sensor rpm-probe {
+                    path test-name;
+                }
+                type string;
+                description "RPM probe test name";
+            }
+            /*
+             * Anomaly detection logic.
+             */
+            trigger rpm-probe-packet-loss-netconf {
+                frequency 90s;
+                /*
+                 * Sets color to red and sends out an anomaly notification when
+                 * RTT packet loss percentage is 1 or above.
+                 */
+                term is-packet-loss-observed {
+                    when {
+                        greater-than-or-equal-to "$loss-percentage" 1;
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "RTT packet Loss percentage is $loss-percentage for $owner, $test-name";
+                        }
+                    }
+                }
+                /*
+                 * Defaults color to green.
+                 */
+                term no-packet-loss {
+                    then {
+                        status {
+                            color green;
+                            message "RTT packet Loss is not observed for $owner, $test-name";
+                        }
+                    }
+                }
+            }
+            trigger rpm-probe-round-trip-time-netconf {
```

Comment:
No need of ingest type in trigger name. Please remove -netconf.

Reply:
modified the trigger name